### PR TITLE
7.x wcag

### PR DESCRIPTION
**Pull request by MorganDawe**
**JIRA Ticket**: (https://jira.duraspace.org/browse/ISLANDORA-2217)

#WCAG Islandora Solr Search

aXe accessibility engine: https://www.deque.com/axe/
Web Accessibility initiative: https://www.w3.org/WAI/intro/wcag

# What does this Pull Request do?
The goal of this pull request is to bring islandora solr search into compliance with WCAG 2.0 standards.

It is important to note that this pull aims to address accessibility violations as seen by the average, non authenticated users. Administrative pages remain unaffected.

# What's new?
Adds the 'aria-label' attribute to the slider handles for use with screen readers and accessibility compliance.

# How should this be tested?

The easiest way to test these changes locally:
- install the aXe browser plugin in your browser of choice (found here: https://www.deque.com/axe/).
- Recommended: Enable the ‘AT Subtheme’ found in the Adaptive Core (https://www.drupal.org/project/adaptivetheme). This theme is the closest to matching wcag 2.0 requirements i could find.
- Run the aXe tool on a person or department page, or another wcag 2.0 compliance testing toolkit, and examine the page for WCAG 2.0 violations sourced from this modules code.

# Interested parties
Tag (@whikloj @Islandora/7-x-1-x-committers)

### CODE REVIEW

**Comment by DonRichards** [comment]
Thank you for this!

File: includes/blocks.inc [hunk 1]
```diff
@@ -623,6 +623,8 @@ function islandora_solr_advanced_search_form($form, &$form_state) {
     $term['field'] = array(
       '#title' => t('Field'),
       '#type' => 'select',
+      '#title' => t('Field'),
```
Comment (rosiel) [inline includes/blocks.inc]: duplicated #title elements?

Comment (MorganDawe) [reply includes/blocks.inc]: Woopsie doo. My mistake.

**Review by willtp87 — APPROVED** [review]

**Comment by MorganDawe** [comment]
@whikloj @Islandora/7-x-1-x-committers Any further suggestions?

**Comment by whikloj** [comment]
@MorganDawe sorry I've been swamped. I'll try to pull in this and a couple other PRs to run through a quick check. But I don't really have any concerns about it. So if @rosiel is good, then she can merge at will.

**Comment by whikloj** [comment]
No changes to function and adds the labels.

**Pull request merged**